packages/odyssey-react/src/components/Banner/Banner.test.tsx

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import Banner from ".";
+
+const role = "status";
+const bannerTitle = "Banner title";
+const bannerContent = "Banner content";
+const bannerActions = "Banner.Children, <a href='https://www.okta.com'>with a link</a>";
```

Comment:
whats with the string that looks like a component here `Banner.Children`? sorta confusing if I'm following what you're doing here

Reply:
This is confusing. Will update.